source/_integrations/withings.markdown

```diff
@@ -16,7 +16,16 @@ The `withings` sensor platform consumes data from various health products produc
 
 ## Setup
 
-### Step 1 - Create a Withings Account
+### Step 1 - Setup Home Assistant to be publicly accessible
+
+- Make Home Assistant publicly accessible over HTTPS a fully qualified domain name.
+  - Subscribing to Home Assistant Cloud (https://www.nabucasa.com/) is a simple way of doing this.
+- Configure Home Assistant with the correct external URL.
+  - In Home Assistant goto Configuration > General > External URL.
+  - Provide the publicly accessible URL.
+  - If you are using Home Assistant Cloud, enter https://YOUR_NABU_CASA_ID.ui.nabu.casa
```

Comment:
This is incorrect. The Nabu Casa URL should never be set here.
It is automatically considered when there is no external URL set.